app/src/org/commcare/UpdateToPrompt.java

```diff
@@ -0,0 +1,158 @@
+package org.commcare;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.util.Base64;
+
+import org.javarosa.core.model.utils.DateUtils;
+import org.javarosa.core.util.externalizable.DeserializationException;
+import org.javarosa.core.util.externalizable.ExtUtil;
+import org.javarosa.core.util.externalizable.Externalizable;
+import org.javarosa.core.util.externalizable.PrototypeFactory;
+
+import java.io.ByteArrayOutputStream;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.util.Date;
+
+/**
+ * Created by amstone326 on 4/13/17.
+ */
+
+public class UpdateToPrompt implements Externalizable {
+
+    public static final String KEY_CCZ_UPDATE_TO_PROMPT = "ccz-update-to-prompt";
+    public static final String KEY_APK_UPDATE_TO_PROMPT = "apk-update-to-prompt";
+
+    private String versionString;
+    private int cczVersion;
```

Comment:
Lol. Fair but definitely not going to change it because ccVersion could easily be misinterpreted to mean "CommCare version" which is not what that value represents.